apps/src/lab2/views/dialogs/dsco/StartOverDialog.tsx

```diff
@@ -0,0 +1,36 @@
+import Modal from '@code-dot-org/component-library/modal';
+import React from 'react';
+
+import {commonI18n} from '@cdo/apps/types/locale';
+
+export type MessageType = 'text' | 'blocks' | 'custom';
+
+interface StartOverDialogProps {
+  onConfirm: () => void;
+  onCancel: () => void;
+  type: MessageType;
+  message?: string;
+}
+
+const StartOverDialog: React.FC<StartOverDialogProps> = ({
```

Comment:
Mentioned this in the description but for @molly-moen @breville

> I also created a new StartOverDialog that uses the DSCO Modal. I realize this is a bit redundant because we already have a dialog system and an existing StartOverDialog. However that dialog system was written pre-DSCO and has some extra complexity that I think we no longer need; moving to DSCO dialogs in general I think will help clean up our shared dialog system and clean up lab code. Happy to discuss further though!